ddtrace/ddtrace.go

```diff
@@ -79,6 +79,48 @@ type SpanContext interface {
 	ForeachBaggageItem(handler func(k, v string) bool)
 }
 
+// ReadableSpan specifies methods to read from a span.
+type ReadableSpan interface {
```

Comment:
Technically this doesn't need to be exported, but I made it exported for the purpose of the documentation of the getters.

Reply:
Perhaps this interface isn't needed and we can just embed the methods into the `ReadWriteSpan` instead?

Reply:
Makes sense to me! Changed 👍